web_services/foxx/brightid/index.js

```diff
@@ -235,8 +235,8 @@ const handlers = {
       }
     }
 
-    if (!db.isVerifiedFor(user, app, verification)) {
-      throw new errors.NotVerifiedError(contextId, appKey);
+    if (verification?.length !== 0 && !db.isVerifiedFor(user, verification)) {
```

Comment:
this should be:

`if (!db.isVerifiedFor(user, verification || app.verification)) {`